src/overlays/actors/ovl_En_Torch2/z_en_torch2.c

```diff
@@ -10,12 +10,51 @@
 
 #define THIS ((Player*)thisx)
 
+typedef enum {
```

Comment:
Should we rename this to dark link

Reply:
IIRC in the long term we would be renaming everything to use names that made more sense, but I think that's for later.

Reply:
I personally do use the better name for actors in things like temp variables and enums, but in the end its stuff that will be discussed and enforced at a future documentation stage anyway, so its left to the discretion of the person decopmiling for now